src/_cffi_src/commoncrypto/secure_transport.py

```diff
@@ -0,0 +1,315 @@
+# This file is dual licensed under the terms of the Apache License, Version
+# 2.0, and the BSD License. See the LICENSE file in the root of this repository
+# for complete details.
+
+from __future__ import absolute_import, division, print_function
+
+INCLUDES = """
+#include <Security/SecureTransport.h>
+"""
+
+TYPES = """
+typedef ... *SSLContextRef;
+typedef const void *SSLConnectionRef;
```

Comment:
No.

The purpose of this is to be a plain object that is passed into/out of secure transport, so ideally you'd use it with `ffi.to_handle` and `ffi.from_handle`. If you change this to a `...` typedef, CFFI complains:

```
Traceback (most recent call last):
  File "/Users/cory/tmp/securetransportscratch/src/securetransport/low_level.py", line 51, in python_write_func
    context = ffi.from_handle(connection)
TypeError: expected a 'cdata' object with a 'void *' out of new_handle(), got 'struct $$SSLConnectionRef *'
```